File: BrianLee/Week3/2022-09-30-Reverse-Bits.java
public class Solution {
    public int reverseBits(int n) {
        int res = 0;
        int bit = 0;
        for(int i = 0; i < 32; i++) {
            bit = (n >> i) & 1;
            res = res | (bit << (31 - i));
        }
        return res;
    }
}

File: DongwanKim/2022-09-30-ReverseBits.js
var reverseBits = function(n) {
    let ans = 0;
    
    for(let i=0; i<32; i++){
        let bit = (n >>> i) & 1;
        ans = (ans | (bit << (31-i)) >>> 0) >>> 0

    }
    return ans
};